packages/medusa-plugin-meilisearch/src/utils/transformer.ts

```diff
@@ -0,0 +1,52 @@
+const variantKeys = [
```

Comment:
suggestion: It looks exactly like the previous one, should we move that to the utils package in the search domain?